charts/dsp-atlantis/templates/_helpers.tpl

```diff
@@ -0,0 +1,11 @@
+{{/*
+Create labels to use for resources in this chart
+*/}}
+{{- define "dsp-atlantis.labels" }}
+    helm.sh/chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    app.kubernetes.io/name: {{ .Chart.Name }}
+    app.kubernetes.io/instance: {{ .Release.Name | quote }}
+    app.kubernetes.io/managed-by: {{ .Release.Service | quote }}
+    app.kubernetes.io/component: atlantis
+    app.kubernetes.io/part-of: dsp-tools
```

Comment:
These get applied automatically to resources created by the base atlantis chart as well?

Reply:
No, they have their own labels. I don't think there's an easy way to apply the labels to everything without forking the official Atlantis chart.

Reply:
yeah that's what I thought, just checking. Oh well.